models/workspacemanager/workspace_provider.py

```diff
@@ -2,7 +2,7 @@
 
 class WorkspaceProvider:
     @abc.abstractmethod
-    def getWorkspaceNames(self):
+    def mtd_getObjectNames(self):
```

Comment:
@OwenArnold  I made the names case compliant and abandoned mimicking the underlying algorithm's casing